react/test/components/NavBarSpec.js

```diff
@@ -0,0 +1,21 @@
+import NavBar from '../../src/components/NavBar';
+
+describe('NavBar', ()=> {
+  let wrapper;
+
+    beforeEach(() => {
+      wrapper = mount(
+        <NavBar
+        />
+      )
+    });
+
+  it('should return true', () => {
+    expect(true).toEqual(true)
+  })
+
+  it('should render a div tag', () => {
+    expect(wrapper.find("div")).toBePresent()
+  })
+
+})
```

Comment:
Your NavBar doesnt do anything right now. There is no feature here, so delete this and do it again later when the NavBar has functionality!